Doc/library/sys.rst

```diff
@@ -1185,6 +1185,9 @@ always available. Unless explicitly noted otherwise, all variables are read-only
    ``cache_tag`` is set to ``None``, it indicates that module caching should
    be disabled.
 
+   *supports_isolated_interpreters* is a boolean value, whether
+   this implementation supports :pep:`734`. It is always ``True`` for CPython.
```

Comment:
Two corrections:

* "supports PEP 684"
* It is False for some platforms, such as WASI (see gh-135190)

I'd also recommend one extra note about the low-level module.

```suggestion
   *supports_isolated_interpreters* is a boolean value, whether
   this implementation supports multiple isolated interpreters.
   It is ``True`` for CPython on most platforms.  Platforms with
   this support implement the low-level :mod:`!_interpreters` module.
   
   .. seealso:
       :pep:`684`, :pep:`734`, and :mod:`concurrent.interpreters`.
```